This is my task

AnasSakka opened the pull request.

AnasSakka committed:
first commit

AnasSakka committed:
the project tasks

AnasSakka committed:
the changes

AnasSakka committed:
enhansment

hanza93 commented:
hey @AnasSakka, it's not recommended to push your vendor folder, please let's add to `.gitignore`

AnasSakka committed:
remove the vendor file

AnasSakka committed:
Add vendor directory and Laravel-specific entries to .gitignore

AnasSakka commented:
> hey @AnasSakka, it's not recommended to push your vendor folder, please let's add to `.gitignore`

I added the vendor folder to ensure the project works successfully on your device. I’ve removed it now."
